kythe/javatests/com/google/devtools/kythe/analyzers/java/testdata/pkg/proto/testdata.proto

```diff
@@ -24,4 +24,11 @@ message Message {
     //- @nested_bool defines/binding NestedBool
     bool nested_bool = 2;
   }
+
+  //- @oneof_field defines/binding OneofField?
```

Comment:
Remove the `?` ?

Reply:
This proto is used in both the java and c++ tests. Without the `?`, the verifier fails on the java tests because it isn't referenced there. I'll send a PR to protobuf team soon for the java annotations, but they don't exist yet.

Reply:
Add `check_for_singletons=False` to the Java tests

Reply:
:+1: